shared/style/date_selector.css

```diff
@@ -277,3 +280,53 @@ button[disabled]::-moz-selection {
 [role="dialog"][data-type="date-selector"] menu button.full {
   width: 100%;
 }
+
+/* Right-To-Left layout */
+html[dir="rtl"] [role="dialog"][data-type="date-selector"] .picker-bar-background {
+  left: 0;
+  right: auto;
```

Comment:
in this case, please use `unset` instead of `auto` (same for the following lines)